packages/launcher/src/main/rpc/app.js

```diff
@@ -270,9 +280,9 @@ export const sandboxed = {
             { mode: 'content-hash' },
           )
           ctx.storage.contentHash = contentHash
-          const list = await ctx.bzz.list(contentHash)
-          if (list.entries) {
-            entries = list.entries.map(meta => {
+          const response = await ctx.bzz.list(contentHash)
+          if (response.entries instanceof Array) {
```

Comment:
Does `instanceof Array` work as expected here? AFAIK `Array.isArray()` should be used to type-check arrays.

Reply:
It does, but that's even better, thanks :+1:

Reply:
Hm, flow complains if I use that check instead:

```
$ flow check --merge-timeout 0
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ packages/launcher/src/main/rpc/app.js:286:34

Cannot call await with ctx.bzz.list(...) bound to p because property entries of unknown type [1] is incompatible with
ListEntry [2].

     packages/launcher/src/main/rpc/app.js
     283│             { mode: 'content-hash' },
     284│           )
     285│           ctx.storage.contentHash = contentHash
     286│           const response = await ctx.bzz.list(contentHash)
 [1] 287│           if (Array.isArray(response.entries)) {
     288│             entries = response.entries.map(meta => {
     289│               return { contentType: meta['contentType'], key: meta['path'] }

     node_modules/@erebos/api-bzz-base/cjs/types.js.flow
 [2]  26│   entries?: Array<ListEntry>,
```